src/firmware-update.html

```diff
@@ -15,7 +15,10 @@ <h1>Firmware update</h1>
   </div>
   <div><h3 class="text-left">SL Web Tools to flash firmware updates from the browser</h3></div>
   <p>
-    If you want to have the latest update now: you can use the web-flasher below to install the latest firmware version for Home Assistant Connect&nbsp;ZBT-1 directly from your browser over USB. You can also use the web-flasher to flash the Zigbee firmware again if you've previously used the experimental multi-protocol support.
+    This page is for troubleshooting. If you use Connect&nbsp;ZBT-1 in Home Assistant, the Home Assistant discovery flow will automatically install the correct firmware. You do not need to install firmware using this tool. Furthermore, on every start of the OpenThread Border router, the latest Thread firmware which is bundled with the add-on will be installed.
+  </p>
+  <p>
+    If for some reason you want to have the latest update now: you can use the web-flasher below to install the latest firmware version for Home Assistant Connect&nbsp;ZBT-1 directly from your browser over USB. You can also use the web-flasher to flash the Zigbee firmware again if you've previously used the experimental multi-protocol support.
```

Comment:
If folks follow the uninstall steps here, flashing when coming from multi-protocol is not necessary. So I think we can remove this part as well.

```suggestion
    If for some reason you like to flash a firmware still: you can use the web-flasher below to install the latest firmware version for Home Assistant Connect&nbsp;ZBT-1 directly from your browser over USB.
```